vars/withCamundaOnlyPipeline.groovy

```diff
@@ -33,6 +33,8 @@ def call(type, String product, String component, String s2sServiceName, String t
     angular: new AngularPipelineType(this, deploymentProduct, component)
   ]
 
+  def deployToProdFlag = ['wa'].contains(product)?false: true;
```

Comment:
Hi, can you look at this line again. Looks overly complicated - would `!product.equals('wa')` do

Reply:
done